content/applications/inventory_and_mrp/purchase/manage_deals/manage.rst

```diff
@@ -2,226 +2,249 @@
+:abbr:`RFQ (request for quotation)`. Doing so reveals a blank :abbr:`RFQ (request for quotation)`
+detail form.
 
-The first thing you will need to do upon creating a vendor bill is to
-select the appropriate vendor, since doing so will also pull up any associated
-accounting or pricelist information. From there, you can choose to
-specify one or multiple Purchase Orders to populate the Vendor Bill
-with. When you select a Purchase Order from the list, Odoo will pull any
-uninvoiced products associated to that Purchase Order and automatically
-populate that information below. If you are having a hard time finding
-the appropriate vendor bill, you may search through the list by
-inputting the vendor reference number or your internal purchase order
-number.
+On the blank detail form, add a vendor to the :abbr:`RFQ (request for quotation)` in the
+:guilabel:`Vendor` field, and add products to the :guilabel:`Product` lines by clicking
+:guilabel:`Add a line`.
 
-.. image:: manage/manage07.png
-  :align: center
+Then, confirm the :abbr:`RFQ (request for quotation)` by clicking the :guilabel:`Confirm Order`
+button above the detail form. Doing so turns the :abbr:`RFQ (request for quotation)` into a purchase
+order.
 
-While the invoice is in a draft state, you can make any modifications you
-need (i.e. remove or add product lines, modify quantities, and change
-prices).
+Finally, click the :guilabel:`Create Bill` button to create a bill for the purchase order.
 
 .. note::
-   Your vendor may send you several bills for the same Purchase Order if:
+   Clicking :guilabel:`Create Bill` before any products have been received will cause a
+   :guilabel:`User Error` pop-up to appear. The :guilabel:`Purchase Order` requires the receipt of
+   at least partial quantity of the items included on the order to create a vendor bill.
 
-   #. Your vendor is in back-order and is sending you invoices as they ship the products.
-   #. Your vendor is sending you a partial bill or asking for a deposit.
+.. image:: manage/manage-user-error-popup.png
+   :align: center
+   :alt: User error pop-up for received quantities control policy.
 
-Every time you record a new vendor bill, Odoo will automatically
-populate the product quantities based on what has been received from the
-vendor. If this value is showing a zero, this means that you have not
-yet received this product and simply serves as a reminder that the
-product is not in hand and you may need to inquire further into this. At
-any point in time, before you validate the Vendor Bill, you may override
-this zero quantity.
+Next, click the :guilabel:`Receipt` smart button to view the warehouse receipt form.
 
-Vendor Bill Matching
-====================
+On the warehouse receipt form, click :menuselection:`Validate --> Apply` to mark the
+:guilabel:`Done` quantities. Then, navigate back to the :guilabel:`Purchase Order` (via the
```

Comment:
Arguable instance of whether or not to use a :guilabel: or :menuselection: here, I'd prefer a menuselection. @brse-odoo @StraubCreative  WDYT? 
```suggestion
:guilabel:`Done` quantities. Then, navigate back to the :menuselection:`Purchase Order` (via the
```

Reply:
I went with the menuselection, I agree it makes sense here